test/Project.toml

```diff
@@ -13,13 +13,15 @@ Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 OSQP = "ab2f91bb-94b4-55e3-9ba0-7f65df51de79"
 ParameterJuMP = "774612a8-9878-5177-865a-ca53ae2495f9"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 PowerModels = "c36e90e8-916a-50a6-bd94-075b64ef4655"
+PowerSimulations = "e690365d-45e2-57bb-ac84-44ba829e73c4"
```

Comment:
Doesn't this break the CI testing? @daniel-thom told me not to add PSI here.

Reply:
Correct. This should not be here.  When you run `]test` PSI gets added implicitly.

Reply:
This was an oversight when I updated all the other toml files.